css/dir.css

```diff
@@ -10,6 +10,10 @@ html {
     background-color: var(--light);
 }
 
+.title.is-4 a{
```

Comment:
Do we need this underline @JoeyDP ? It looks a bit out of place imo. I like what Olivier did for the papers, if you hover it, it changes colour, indicating it's a link.

Reply:
@LienM, changed in new commit. Because they are bold, it looks a bit out of place maybe? Feel free to suggest other edits.